docs/en_US/bridges/aws-iot-core-bridge.md

```diff
@@ -126,12 +126,12 @@ bridges.aws.c1 {
 	subscription = [
 		{
 			remote_topic = "cmd/topic1"
```

Comment:
cmd/topic3

Reply:
I don't think it's a problem, remote_topic and local_topic are not strongly correlated.